# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hyprcursor/hotfix-fstream.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hyprcursor-0.1.10/hyprcursor-util/src/main.cpp.vanilla	2025-01-27 17:26:24.895076786 +0100
+++ hyprcursor-0.1.10/hyprcursor-util/src/main.cpp	2025-01-27 17:27:21.138314034 +0100
@@ -7,6 +7,7 @@
 #include <algorithm>
 #include <regex>
 #include <hyprlang.hpp>
+#include <fstream>
 #include "internalSharedTypes.hpp"
 #include "manifest.hpp"
 #include "meta.hpp"
